src/utils/suggestions.js

```diff
@@ -62,7 +66,7 @@ const getSuggestions = (fields, suggestions, currentValue, suggestionProperties
 		}
 	};
 
-	const parseField = (source, field) => {
+	const parseField = (source, field, id) => {
```

Comment:
Please refer the source from here instead of creating a getSourceValue function?